### PR TITLE
Improve performance for 5G SCM

### PR DESCRIPTION
**Pull request by eric-intuitem**
It is sufficient to show them in each requirement assessment.

### CODE REVIEW

File: backend/core/helpers.py [hunk 1]
```diff
@@ -309,12 +309,6 @@ def get_sorted_requirement_nodes_rec(
                             "status_display": req_as.get_status_display(),
                             "status_i18n": camel_case(req_as.status),
                             "style": "leaf",
-                            "threats": ThreatReadSerializer(
-                                req.threats.all(), many=True
-                            ).data,
-                            "reference_controls": ReferenceControlReadSerializer(
-                                req.reference_controls.all(), many=True
-                            ).data,
```
Comment (nas-tabchiche) [inline backend/core/helpers.py:317]: Doesn't this break the display of threats and reference controls in the requirement trees?

Comment (ab-smith) [reply backend/core/helpers.py:317]: The idea is to take them out of the tree and keep them only on the requirement assessment to accelerate the tree building

Comment (eric-intuitem) [reply backend/core/helpers.py:317]: It simply removes them, and it's way faster to get the display.

**Review by ab-smith — APPROVED** [review]

File: backend/core/helpers.py (hunk 1 above)
Comment (nas-tabchiche) [reply backend/core/helpers.py:317]: Nothing that can't be fixed with prefetching. I am aware this branch was merged, but are we really sure we don't want to keep suggested threats/applied controls in the tree?

**Pull request merged**